RELEASE 2014.12 unable to build from download

The release 2014.12 fails to build any applications for any board when building from an extracted archive (instead of git).

Here is the error:

    make BOARD=native all 
    Building application "rpl_udp" for "native" with MCU "native".
    /bin/sh: 1: Syntax error: "(" unexpected
    make[1]: ***     [/home/xsoen/Documents/Developpement/workspace/RIOT_Release_2014_dec/examples/rpl_udp/bin/native/rpl_udp/helper.o] Erreur 2
    make: *** [all] Erreur 2

--- a/Makefile.include
+++ b/Makefile.include
@@ -130,7 +130,7 @@ ifeq ($(origin RIOT_VERSION), undefined)
       RIOT_VERSION := $(GIT_STRING)-$(GIT_BRANCH)
     endif
   else
-    RIOT_VERSION := UNKNOWN (builddir: $(RIOTBASE))
+    RIOT_VERSION := "UNKNOWN (builddir: $(RIOTBASE))"
   endif
 endif
 export CFLAGS += -DRIOT_VERSION=\"$(RIOT_VERSION)\"

